test/acr_cloud_response_test.dart

```diff
@@ -0,0 +1,216 @@
+import 'package:flutter_acrcloud/src/acrcloud_response.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  test(
+    'Should parse music JSON metadata',
```

Comment:
super nit: I'm not a huge fan of starting every test name with "should". I would just call these tests "parses music JSON metadata" and "parses custom files JSON metadata".

Thanks for adding tests by the way!